src/main/scala/mesosphere/marathon/state/PathId.scala

```diff
@@ -221,7 +208,7 @@ object PathId {
     id.path.forall(part => ID_PATH_SEGMENT_PATTERN.pattern.matcher(part).matches())
   }
 
-  private val reservedKeywords = Seq("restart", "tasks", "versions")
+  private val reservedKeywords = Seq("restart", "tasks", "versions", ".", "..")
```

Comment:
This shouldn't be an issue, right? The reserved keywords are checked for the last element, and we certainly don't want '.' or '..' as last element?

Reply:
We certainly shouldn't have it as the last element but it is totally valid to have those keywords in relative paths

Reply:
Yes, inside the pathes it's allowed, the reserved keywords are only checked for the last element.